python/lsst/faro/measurement/TractTableMeasurementTasks.py

```diff
@@ -144,3 +146,93 @@ def run(
                 metricName, np.mean(np.abs(result["corr"])), extras=extras
             )
         )
+
+
+class wPerpTableConfig(Config):
+    """Class to organize the yaml configuration parameters to be passed to
+    TExTableTask when using a parquet table input. All values needed to perform
+    TExTableTask have default values set below.
+
+    Optional Input (yaml file)
+    ----------
+    Column names specified as str in yaml configuration for TeX task. These are
+    the desired column names to be passed to the calcuation. If you wish to use
+    values other than the default values specified below, add the following e.g.
+    line to the yaml file:
+
+    config.measure.raColumn = "coord_ra_new"
+    """
+
+    bright_rmag_cut  = Field(
```

Comment:
Two spaces here

Reply:
Not sure why you were seeing that line -- it is not on this ticket branch.

Reply:
It's because it's in the commit history for this PR and I'm guessing @sr525 did a commit-by-commit review.

Reply:
See commit 68ad04d2eff4815aa1aa52b413a233d073d85149

Reply:
Yep, there's no reason to review each commit.